tests/test_schemes.py

```diff
@@ -99,6 +101,19 @@ def test_failed_connection(self):
         scheme = Scheme('wlan0', 'test')
         self.assertRaises(ConnectionError, scheme.parse_ifup_output, FAILED_IFUP_OUTPUT)
 
+    def test_activate_is_called_with_good_args(self):
+        args = ['sudo', '/sbin/ifdown', 'wlan0', 'wlan0=wlan0-test']
+        kwargs = {'stderr':subprocess.STDOUT}
+        scheme = Scheme('wlan0', 'test')
+        subprocess.check_output = MagicMock(return_value=SUCCESSFUL_IFUP_OUTPUT)
```

Comment:
I don't think that you removed the mock after this, so it's possible that subprocess.check_output is now mocked for all the other tests.

You can use the `patch` function from mock as a context manager, I think that that would be cleaner.

Thoughts?